code/modules/recycling/conveyor2.dm

```diff
@@ -105,15 +106,12 @@
 	use_power(100)
 
 	affecting = loc.contents - src		// moved items will be all in loc
-	sleep(1)	// slight delay to prevent infinite propagation due to map order
-	var/items_moved = 0
+	sleep(1)
 	for(var/atom/movable/A in affecting)
 		if(!A.anchored)
 			if(A.loc == src.loc) // prevents the object from being affected if it's not currently here.
 				step(A,movedir)
-				items_moved++
-		if(items_moved >= 10)

```

Comment:
Mostly a question - not sure if this is really a problem (maybe you already know or it's fine), but with removed limit i see a constant CPU usage - https://dl.dropboxusercontent.com/u/73967379/A344DDH3dqqPP9--i/dasd32sada.png - 1000(!) PDAs trying to move into the wall, which makes high CPU usage (local server with no players). But yeah, 1k atoms... Probably not an issue for normal rounds and only one scenario in my mind: someone blocks conveyor belt and then drops huge amount of items until it starts to eat alot of CPU and slowing down everything else or no?